lte/gateway/c/session_manager/LocalEnforcer.h

```diff
@@ -458,7 +458,8 @@ class LocalEnforcer {
    */
   void propagate_rule_updates_to_pipelined(
       const SessionConfig& config, const RulesToProcess& pending_activation,
-      const RulesToProcess& pending_deactivation, bool always_send_activate);
+      const RulesToProcess& pending_deactivation, bool always_send_activate,
+      uint16_t shard_id);
```

Comment:
nit: the order looks off, the parameters that identify the session/UE should come first and then information about what/how to propagate
```
void propagate_rule_updates_to_pipelined(
        const SessionConfig& config, uint16_t shard_id, 
        const RulesToProcess& pending_activation,
        const RulesToProcess& pending_deactivation, bool always_send_activate);
```